### PR TITLE
28090 accordion uplift

### PR DESCRIPTION
**Pull request by jen-lui**

### CODE REVIEW

**Review by uqnm — APPROVED** [review]
looks good

**Review by Bcrunk — COMMENTED** [review]
Should we still specify a default? 

We could still telegraph to users the preferred implementation. 
i.e. if we think the best way of doing it is with lists? (which ever is most accessible and machine radable)

See the "Buttons" example .

**Review by Bcrunk — COMMENTED** [review]
not sure about the Alternate behaviour being part of how and when to use. 
 
The alternate behaviour is still just a variant and may be better placed there. 

Cards as an example still has everything as a variant.. How and when to use is more about the component usage and less about the variations in functionality?

**Review by Bcrunk — COMMENTED** [review]
The naming still throws me off, is-manual feel wrong.
I took a look at Material design, they have the functionality fliped but refer to the options as 

Collapsible and Expandable 

I am leaning more towards is-expandable (just my opinion / recommendation, not a red flag)

**Review by Bcrunk — CHANGES_REQUESTED** [review]

**Comment by jen-lui** [comment]
> Should we still specify a default?
> 
> We could still telegraph to users the preferred implementation.
> i.e. if we think the best way of doing it is with lists? (which ever is most accessible and machine radable)
> 
> See the "Buttons" example .

For this component, there isn't really a default because the docs provide guidance on which to use depending on the content that needs to be displayed.

**Comment by jen-lui** [comment]
> not sure about the Alternate behaviour being part of how and when to use.
> 
> The alternate behaviour is still just a variant and may be better placed there.
> 
> Cards as an example still has everything as a variant.. How and when to use is more about the component usage and less about the variations in functionality?

Agree, have moved this from "How and when to use" to a separate section for behaviour and the variant.

**Comment by jen-lui** [comment]
> The naming still throws me off, is-manual feel wrong.
> I took a look at Material design, they have the functionality fliped but refer to the options as
> 
> Collapsible and Expandable
> 
> I am leaning more towards is-expandable (just my opinion / recommendation, not a red flag)

is-expandable feels odd because all of the accordions are expandable? I've updated the docs so hopefully the distinction is more obvious between default automatic expand/collpase behaviour and variant manual expand/collapse behaviour.

File: packages/accordion/src/scss/_component.scss [hunk 1]
```diff
@@ -42,43 +38,50 @@
     cursor: pointer;
     display: inline-block;
     font-family: core.$font-body;
-    font-size: 1.2rem;
+    font-size: 1.25rem;
```
[the remark this answers is not in the document]
Comment (jen-lui) [reply packages/accordion/src/scss/_component.scss]: Fixed

**Review by Bcrunk — APPROVED** [review]

**Pull request merged**